src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -11672,6 +11693,17 @@ void Assembler::evbroadcasti64x2(XMMRegister dst, Address src, int vector_len) {
   emit_operand(dst, src, 0);
 }
 
+void Assembler::vbroadcasti128(XMMRegister dst, Address src) {
```

Comment:
Please see vbroadcastf128, good to take vector_len as argument here also and have an assert to confirm that it is AVX-256 bit. 
Also need:
attributes.set_address_attributes(/* tuple_type */ EVEX_T4, /* input_size_in_bits */ EVEX_32bit);